packages/client/hmi-client/src/workflow/ops/model-optimize/tera-model-optimize-node.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+	<section>
+		<img src="@assets/svg/plants.svg" alt="" draggable="false" />
+		Configure in the side panel
+	</section>
```

Comment:
We have a [tera-operator-placeholder-graphic](https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/workflow/operator/tera-operator-placeholder-graphic.vue) for that purpose.

And we do not have a side panel anymore, we open the drilldown.